city_scrapers/spiders/il_opioid_response.py

```diff
@@ -0,0 +1,144 @@
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlOpioidResponseSpider(CityScrapersSpider):
+    name = "il_opioid_response"
+    agency = "Illinois Opioid Crisis Response Advisory Council"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.dhs.state.il.us"]
+    start_urls = ["https://www.dhs.state.il.us/page.aspx?item=104682"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+
+        for item in response.xpath('//p//a'):
+            if 'Agenda' not in item.xpath('text()').get():
+                break
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item, response),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
```

Comment:
This should have a default like “Advisory Council”, or a unique meeting name if it has one (doesn’t seem like this does though)